client/verta/tests/test_permissions/test_workspace.py

```diff
@@ -61,25 +61,24 @@ def test_model_ver(self, client, organization, created_entities):
         )
 
         # organization workspace
-        reg_model = client.create_registered_model(workspace=organization.name)
+        reg_model = client.create_registered_model(workspace=workspace.name)
         created_entities.append(reg_model)
-        assert reg_model.create_version().workspace == organization.name
-
+        assert reg_model.create_version().workspace == workspace.name
 
 class TestClientGetWorkspace:
     """Order of precedence for `client.get_workspace()`."""
 
     def test_client_get_workspace(
-        self, create_client, create_organization, created_entities, in_tempdir
+            self, create_client, created_entities, in_tempdir, workspace
     ):
         client = create_client()
 
         WEBAPP_WORKSPACE = (
             client._conn.get_default_workspace()
         )  # TODO: first change default workspace
-        CONFIG_WORKSPACE = create_organization().name
-        CLIENT_WORKSPACE = create_organization().name
-        PARAM_WORKSPACE = create_organization().name
+        CONFIG_WORKSPACE = workspace.name
+        CLIENT_WORKSPACE = workspace.name
+        PARAM_WORKSPACE = workspace.name
```

Comment:
If possible, these three workspaces need to be different, not the same one.

Reply:
yes, it is possible, but we need workspace creation for that, so I added it now to workspace creation PR

Reply:
👍 Great, I see that it was added in https://github.com/VertaAI/modeldb/pull/3439/commits/33079e293fc9ae1d70ee63bbd8389012e184c6f9